src/Panes/TerminalPane.vala

```diff
@@ -22,6 +22,7 @@ namespace ElementaryTweaks {
         private Gtk.Switch natural_copy_paste;
         private Gtk.Switch follow_last_tab;
         private Gtk.Switch unsafe_paste_alert;
+        private Gtk.Switch rem_tabs;
```

Comment:
You should use the full name :) `remember_tabs` instead, it's not too long and it would be consistent with the rest of the elements